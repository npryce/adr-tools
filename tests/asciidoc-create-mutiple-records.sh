echo "adoc" > .adr-file-type
adr new First Decision
adr new Second Decision
adr new Third Decision
ls doc/adr
head -7 doc/adr/0002-second-decision.adoc
head -7 doc/adr/0003-third-decision.adoc
