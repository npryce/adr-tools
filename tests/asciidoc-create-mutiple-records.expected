echo "adoc" > .adr-file-type
adr new First Decision
doc/adr/0001-first-decision.adoc
adr new Second Decision
doc/adr/0002-second-decision.adoc
adr new Third Decision
doc/adr/0003-third-decision.adoc
ls doc/adr
0001-first-decision.adoc
0002-second-decision.adoc
0003-third-decision.adoc
head -7 doc/adr/0002-second-decision.adoc
= 2. Second Decision

Date: 1992-01-12

== Status

Accepted
head -7 doc/adr/0003-third-decision.adoc
= 3. Third Decision

Date: 1992-01-12

== Status

Accepted
